tests/graph-codegen.test.js

```diff
@@ -0,0 +1,64 @@
+const fs = require('fs')
+const { join } = require('path')
+
+const test = require('ava')
+const { buildSchema, parse } = require('graphql')
+
+const {
+  extractFunctionsFromOperationDoc,
+  generateFunctionsSource,
+  generateHandlerSource,
+} = require('../src/lib/one-graph/netlify-graph')
+
+const { normalize } = require('./utils/snapshots')
+
+const netligraphConfig = {
+  extension: 'mjs',
+  netligraphPath: 'netlify',
+  moduleType: 'commonjs',
+  functionsPath: 'functions',
+  netligraphImplementationFilename: 'dummy/index.mjs',
```

Comment:
This is puiblic-facing, right?

Reply:
The test?

And are you suggesting the name is inappropriate? I could rename it to `placeholder` if that's more appealing.

Reply:
No, `netligraphImplementationFilename`

Reply:
I don't currently expect the user to provide `netligraphImplementationFilename`.

But we should change the toml key from `[netligraph]` to `[graph]`